### ISSUE TITLE
Vhe editor terminal 

### ISSUE THREAD

**Issue by n1lby73** [vhqtvn/VHEditor-Android#251]
is it possible to link tye terminal enviroment of vhe editor to my pre-installed termux app

I'm rooted and have tried using sudo to change directory to termux directory but it returns the below error

```
env: exec cd no such file or directory
```

**Comment by HGRgamer** [comment]
I don't think it is possible without root as it is termux internal storage and can be accessed by only that.

A workaround would be to install code-server in termux and use the Start Remote option in vhe editor to connect to it. It should work exactly same as before.

**Comment by Alan2882828** [comment]
Sem root isso e quase imposivel mais a unica maneira e criar o código no VHeditor e depois entrar no termux manualmente e navegar ate o diretório do código e executar ...
Isso co sua linguagem instalada no termux
Ex: python, nodejs para javascript, .....
Mais hj tem o VHeditor com o termux embutido
Mais com algus bugs ....

**Comment by n1lby73** [comment]
> Sem root isso e quase imposivel mais a unica maneira e criar o código no VHeditor e depois entrar no termux manualmente e navegar ate o diretório do código e executar ...
> Isso co sua linguagem instalada no termux
> Ex: python, nodejs para javascript, .....
> Mais hj tem o VHeditor com o termux embutido
> Mais com algus bugs ....

tried that sometime ago, but couldn't get it to work

**Comment by n1lby73** [comment]
> I don't think it is possible without root as it is termux internal storage and can be accessed by only that.
> 
> A workaround would be to install code-server in termux and use the Start Remote option in vhe editor to connect to it. It should work exactly same as before.

would try it out when i get a new device, and just to be sure; using rooted device should work out of the box with sudo right
